XiEditor/Search.swift

```diff
@@ -15,6 +15,11 @@
 import Cocoa
 import Swift
 
+extension NSColor {
```

Comment:
as an item of personal style I might've just had a `static var fallbackBackgroundGray` on the `FindViewController` class instead of doing this as an extension, but I don't have an argument beyond personal taste, so I think this is fine. :)